### ISSUE TITLE
some questions

### ISSUE THREAD

**Issue by masonwang513** [lyxok1/STM-Training#1]
Hi, there are some questions:
1. why only first 5 frames are used ?
    frames = [name[:5] for name in os.listdir(annofolder)]
2. why is it correct to accumulate losses of several frames and compute gradient just based on last frame?

**Comment by lyxok1** [comment]
According to the computation graph, output mask prediction at each frame will be included into gradient computation, not just the last frame.

**Comment by shoutOutYangJie** [comment]
Hi，Do you add youtube-vos train dataset for training?

**Comment by lyxok1** [comment]
@shoutOutYangJie Yes, Youtube and DAVIS are hybrid to train the segmentation network, but no synthetic sequence from static image is used.
